data/language/dutch.txt

```diff
@@ -3360,7 +3360,7 @@ STR_3358    :Kan het baanontwerp niet verwijderen...
 STR_3359    :{BLACK}Geen baanontwerpen van dit type
 STR_3360    :Waarschuwing!
 STR_3361    :Te veel baanontwerpen van dit type; sommige staan niet in de lijst
-STR_3362    :Mixing via softwarebuffer forceren
+STR_3362    :Mixen via softwarebuffer forceren
```

Comment:
Dan zou ik wel de voorkeur geven aan *Mengen van geluid via softwarebuffer forceren* of *Geluid mengen via softwarebuffer forceren*, anders is het niet duidelijk genoeg dat het om geluid gaat. (Of wordt dit uit de context duidelijk?)